core/src/KokkosExp_MDRangePolicy.hpp

```diff
@@ -432,6 +490,36 @@ void md_parallel_for( MDRange const& range
   closure.execute();
 }
 #endif
+// ROCm specialization
```

Comment:
Remove this dead code.

Reply:
will fix

Reply:
I had that in because there was an equivalent cuda section.  I guess that will eventually also be removed.